Rakefile

```diff
@@ -1,9 +1,9 @@
-version_string = `git describe --tag | cut  -d "-" -f 1,2 | tr - .`.chomp
-if version_string.empty?
-  version_string = '0'
+# Attempt to grab the date of the latest commit. If unable to, we use the current date.
```

Comment:
From the Conversation note:

> I updated how we retrieve the date of publication. I now do it by the last git commit date -- this ensures that when we checkout a tag, we can run it, and it gives us consistent dates.

I know we have added this to the release process but how are we ensuring that the latest commit. Will there be a situation where the latest commit is not corresponding to a tag, i.e., changes not released yet?

Reply:
Yes, intentionally. So, when we automatically release to GH pages, it'll grab the tip-of-head's date. However, when we release a version, the exact commit will be tagged. Checking out that commit and running this will always give us the release date matching that of the tagged commit.

Reply:
Just to clarify further: so the page that we will see in the website is not necessarily the released version? if so, would it make sense to automate the release and as part of that publish for website?